testsuite/tests/lib-bigarray/bigarrays.ml

```diff
@@ -150,6 +150,14 @@ let _ =
                   Complex.i, Complex.i;
                   {im=0.5;re= -2.0}, {im=0.5;re= -2.0};
                   {im=3.1415;re=1.2345678}, {im=3.1415;re=1.2345678}]);
+  test 20 true
+    (test_setget int_least31
+                 [0, 0;
+                  123, 123;
+                  -456, -456;
+                  max_int, max_int land 0xFFFFFFFF;
```

Comment:
Indeed, OCaml's `max_int` is 31-bits maxint, so it should be safe. Sorry for the trouble.